content/en/docs/reference/command-line-tools-reference/feature-gates/job-success-policy.md

```diff
@@ -10,5 +10,9 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.30"
+    toVersion: "1.30"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
 ---
 Allow users to specify when a Job can be declared as succeeded based on the set of succeeded pods.
```

Comment:
Maybe we  could change this part in the doc

```
The Job that meets the success policy gets the SuccessCriteriaMet condition
```

into 

```
The Job that meets the success policy gets the SuccessCriteriaMet condition with a SuccessPolicy reason.
```

since we changed the scope of the condition in this release.

Reply:
Great point!
Thank you!

Reply:
Done.